tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -491,9 +491,12 @@ def get_compute_worker_run_info(self, scheduled_run_id: str):
 def test_get_compute_worker_runs(mocker: MockerFixture) -> None:
     client = ApiWorkflowClient(token="123")
     mock_compute_worker_api = mocker.create_autospec(DockerApi, spec_set=True).return_value
-    mock_compute_worker_api.get_docker_runs.return_value = [
-        DockerRunData(id="run-1", created_at=20,  dataset_id="", docker_version="", state="", last_modified_at=0),
-        DockerRunData(id="run-2", created_at=10,  dataset_id="", docker_version="", state="", last_modified_at=0),
+    mock_compute_worker_api.get_docker_runs.side_effect = [
+        [
+            DockerRunData(id="run-1", created_at=20,  dataset_id="", docker_version="", state="", last_modified_at=0),
+            DockerRunData(id="run-2", created_at=10,  dataset_id="", docker_version="", state="", last_modified_at=0),
+        ], 
+        [],
     ]
```

Comment:
If side effect is an iterable then every call to the function (`get_docker_runs`) will return the next value in the iterable. The change was necessary because the endpoint is now paginated and should return multiple lists of runs.